account_credit_control/views/credit_control_line.xml

```diff
@@ -0,0 +1,180 @@
+                        </group>
+                        <group>
+                            <field name="amount_due"/>
+                            <field name="balance_due"/>
+                            <field name="currency_id"/>
+                        </group>
+                    </group>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+    <record id="credit_control_line_search" model="ir.ui.view">
+        <field name="name">Credit Control Lines</field>
+        <field name="model">credit.control.line</field>
+        <field name="arch" type="xml">
+            <search string="Control Credit Lines">
+                <group string="Filters">
+                    <filter name="filter_draft" icon="terp-mail-message-new"
+                            string="Draft" domain="[('state', '=', 'draft')]"
+                            help="Draft lines have to be triaged."/>
+                    <filter name="filter_to_be_sent" icon="terp-mail-forward"
+                            string="Ready To Send"
+                            domain="[('state', '=', 'to_be_sent')]"
+                            help="These lines are ready to send by email or by letter using the Actions."/>
+                    <filter name="filter_ignored" icon="terp-mail_delete"
+                            string="Ignored"
+                            domain="[('state', '=', 'ignored')]"
+                            help="Lines which have been ignored from previous runs."/>
+                    <filter name="filter_sent" icon="terp-mail-replied"
+                            string="Sent" domain="[('state', '=', 'sent')]"
+                            help="Lines already sent."/>
+                    <filter name="filter_error" icon="terp-gtk-stop"
+                            string="Error"
+                            domain="[('state', 'in', ('error', 'email_error'))]"
+                            help="An error has occured during the sending of the email."/>
+                    <filter name="filter_manual" icon="terp-gtk-stop"
+                            string="Manual change"
+                            domain="[('manually_overridden', '=', True)]"
+                            help="The line was deprecated by a manual change of policy on invoice."/>
+                    <separator orientation="vertical"/>
+
+                    <field name="date"/>
+                    <field name="channel"/>
+                    <field name="policy_id"/>
+                </group>
+                <newline/>
+                <group expand="0" string="More...">
+                    <field name="partner_id"/>
+                    <field name="account_id"/>
+                    <field name="invoice_id"/>
+                    <field name="level"/>
+                    <field name="policy_level_id"/>
+                </group>
+
+                <newline/>
+                <group expand="0" string="Group By...">
```

Comment:
Add name selector if you can, as it is recommended at https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-10.0